labshare-client/src/backend/mail/localization.ts

```diff
@@ -6,33 +6,33 @@ export default {
       subject: "LabHive - Account activation",
       text: (link: string) => {
         return `
-Hallo, 
-um Ihren Account auf LabHive zu aktivieren, verwenden Sie bitte folgenden Link:
+Dear user, 
+to activate your account on LabHive, please use the following link:
 ${link}
 
-Ihr LabHive Team
+Best, LabHive Team
```

Comment:
I would close the emails with "Regards" rather than "Best", but that is purely an opinion.